src/unity/lib/unity_sframe.cpp

```diff
@@ -1840,8 +1934,11 @@ void unity_sframe::explore(const std::string& path_to_client, const std::string&
                 }
                 break;
               case flex_type_enum::LIST:
-                ss << value.to<std::string>();
-                break;
+                {
```

Comment:
Shouldn't this whole section (switch/case over type, with stringification) get refactored into `listHelper`? Seems like almost identical logic.